### ISSUE TITLE
Implement nav ui alignment 

### ISSUE THREAD

**Issue by kristinashu** [mozilla/network-pulse#731]
From alignment work in mozilla/network#582 and to align better with new brand, mozilla.org, Network site, and IRH sites.

https://redpen.io/fv1d606e85be529df4

![pulse home 2x](https://user-images.githubusercontent.com/1068783/31301808-b3d6913a-aab1-11e7-95f9-dccaf522372b.png)

### Updates include:
- keep logo the same
- update menu items Open Sans Bold 16px
- search icon the same
- left align nav items and seach
- gradient from #FF506E to #A56EFF | 4px high
- Signin / Signup or Username right aligned #A668FF
- hover state for nav items and Username is black underline 2px high
- Add entry CTA button has hover state and styling similar to https://network.mozilla.org/ but 40x40px, #A668FF with black stock and shadow, icon [icon-add.svg.zip](https://github.com/mozilla/network-pulse/files/1360214/icon-add.svg.zip)
- total nav bar height is 88px
- mobile icons [Icons.zip](https://github.com/mozilla/network-pulse/files/1364105/Icons.zip)

### Mobile:
Similar to https://network.mozilla.org/ 

#### Medium width
<img width="582" alt="screen shot 2017-10-06 at 4 27 19 pm" src="https://user-images.githubusercontent.com/1068783/31302009-467982e4-aab3-11e7-8172-278fddea17b5.png">

#### Narrow width
<img width="770" alt="screen shot 2017-10-06 at 4 41 10 pm" src="https://user-images.githubusercontent.com/1068783/31302266-3ccbcd22-aab5-11e7-8c9e-af5125512fc0.png">

cc @mmmavis

**Comment by xmatthewx** [comment]
@mmmavis – I expect that this work will take a bit of time and focus. In terms of priorities, let's stick to Profile related things first. I discussed with Jesse, and we want to make sure we ship that stuff well first, and then add this UI polish. I don't want you to feel overwhelmed given your other MozFest duties. 

Make sense? Let me know if you have any questions or want to discuss. 

When ready, we should also connect with Gavin to align how we handle the hamburger yum yum. Again, I think you don't need to worry about this unless you're blocked on profile related things. cc @alanmoo

**Comment by mmmavis** [comment]
@kristinashu are we removing "Featured" from nav bar?

**Comment by kristinashu** [comment]
No, you can keep Featured.

**Comment by kristinashu** [comment]
Cross linking this with the Network site nav updates https://github.com/mozilla/network/issues/820

**Comment by xmatthewx** [comment]
When we implement this, let's make sure we remove the old event tracking: https://github.com/mozilla/network-pulse/blob/c0bb8bc04a062623a249ca87186b2fe21992e167/components/nav-link/nav-link.jsx#L13-L14

**Comment by mmmavis** [comment]
@kristinashu this is ready for QA on staging: https://network-pulse-staging.herokuapp.com Things are not pixel perfect as I found some inconsistency in existing elements between Sketch file and production site. Please take a look at the staging site and let me know if anything needs changing. 😃

**Comment by kristinashu** [comment]
Exciting! Thanks @mmmavis!

#### Few updates to make it more like the comp and Foundation site:
- reduce height of the entire header (from 220px to 180px)
- make '+' icon hover state smooth like on Foundation site, rather than sudden jump
- selected state for menu item should be underlined
- link the little 'm' to the Feature page (like the full logo)
- here's the logo in black [pulse-logo.svg.zip](https://github.com/mozilla/network-pulse/files/1471809/pulse-logo.svg.zip)

#### Bug: (maybe open a new ticket for this?)
If open the menu at a narrow screen width and then make it wide, the menu looks like this (seems ok on Foundation site): 
<img width="872" alt="screen shot 2017-11-14 at 8 59 09 am" src="https://user-images.githubusercontent.com/1068783/32793245-3392bf80-c91a-11e7-8948-55145b62a6fb.png">

**Comment by mmmavis** [comment]
@kristinashu 

Tweaks will be up on http://network-pulse-staging.herokuapp.com/ in about 10 mins (currently being deployed)

### Addressed the following items on https://github.com/mozilla/network-pulse/issues/731#issuecomment-344330899

> - reduce height of the entire header (about from 220px to 180px)
> - make '+' icon hover state smooth like on Foundation site, rather than sudden jump
> - selected state for menu item should be underlined
> - link the little 'm' to the Feature page (like the full logo)

### Didn't get addressed

> - here's the logo in black pulse-logo.svg.zip

Not sure what this file is for. Might be for logo hover state? I checked a few Mozilla sites and none of them inverses logo colour on hover state. --- let's discuss this further in a followup ticket.

> Bug: (maybe open a new ticket for this?)
> If open the menu at a narrow screen width and then make it wide, the menu looks like this (seems ok on Foundation site):

This is probably not worth fixing as it is a rare case and there's probably no "clean" way of fixing it (without JS or duplicated DOM elements).

**Comment by kristinashu** [comment]
Awesome! Looks good!
